UnityProject/Assets/Scripts/HealthV2/Living/Mutations/Surface/FireEffect.cs

```diff
@@ -0,0 +1,40 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+namespace HealthV2.Living.Mutations.Surface
+{
+	[CreateAssetMenu(fileName = "FireEffect", menuName = "ScriptableObjects/Mutations/FireEffect")]
+	public class FireEffect : MutationSO
+	{
+		[Tooltip("so, - Would decrease Fire protection max is Double Damage + increases how much Fire protection max is 100% Protection ")]
```

Comment:
tooltip isn't clear enough.
Maybe try my suggestion:
```suggestion
		[Tooltip("Fire protection in percentage. Negative numbers increase incoming damage.")]
```